### PR TITLE
Http Handler Improvement

### PR DESCRIPTION
**Pull request by goru97**
*What: 
Added Headers Parameter in sendResponse Method of DefaultHandler class.

*Why:
To enable extended handlers (like HttpAggregatedIngestionHandler, HttpEventsIngestionHandler etc) send custom headers in response.

### CODE REVIEW

File: blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java [hunk 1]
```diff
@@ -80,25 +80,25 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             List<Boolean> persisteds = futures.get(timeout.getValue(), timeout.getUnit());
             for (Boolean persisted : persisteds) {
                 if (!persisted) {
-                    DefaultHandler.sendResponse(ctx, request, null, HttpResponseStatus.INTERNAL_SERVER_ERROR);
+                    DefaultHandler.sendResponse(ctx, request, null, HttpResponseStatus.INTERNAL_SERVER_ERROR, null);
```
Comment (VinnyQ) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java]: I am not fond of adding a new parameter argument to an existing functions and modifying any current code that needs it to have it call by passing "null".

May I recommend you create a wrapper method also called sendResponse() in the DefaultHandler class, only needing the original parameters, and have that called your modified method with the null as the headers parameter?

Then you can remove all these modification to the endResponse() calls and not have to pass in a "null" when it's not actually needed.

Comment (goru97) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java]: Can't agree more! :)

File: blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java [hunk 2]
```diff
@@ -33,7 +36,18 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
     }
 
     public static void sendResponse(ChannelHandlerContext channel, HttpRequest request, String messageBody, HttpResponseStatus status) {
+        sendResponse(channel, request, messageBody, null);
+    }
```
Comment (VinnyQ) [inline blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java]: NIT:  spacing (need new line).  But wait and see if there's other changes you'll need to make from the other reviewers.

**Comment by VinnyQ** [comment]
:+1: lgtm if all checks are passed.

**Pull request merged**